support for manually specified dependencies (#236)
fossas/spectrometer: .github/pull_request_template.md

## Checklist

- [] I added tests for this PR's change.
- [] I linted (`hlint`) and formatted (`ormolu`) any files I touched in this PR.
- [] If this PR introduced a user-visible change, I added documentation into `docs/`.

> - [] I updated `CHANGELOG.md`. If the PR did not mark a release, update the `#Unreleased section at the top`.

Reviewer: ```suggestion
- [] I updated the `Unreleased` section in `CHANGELOG.md` with a description of my changes
```
